scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -314,6 +323,7 @@ struct task_ctx {
 	struct bpf_cpumask __kptr *layered_cpumask;
 
 	bool			all_cpus_allowed;
+	bool			is_override;
```

Comment:
and then this can be the copy of the generation number where it's only effective if it matches the current gen.